src/main/scala/scalismo/registration/LandmarkRegistration.scala

```diff
@@ -97,16 +97,93 @@ object LandmarkRegistration {
    *
    * @param landmarks sequence of corresponding landmarks
    */
+  def rigid3DLandmarkRegistration(landmarks: Seq[(Point[_3D], Point[_3D])]): RigidTransformation[_3D] = rigid3DLandmarkRegistration(landmarks, origin3D)
 
-  def rigid3DLandmarkRegistration(landmarks: IndexedSeq[(Point[_3D], Point[_3D])]): RigidTransformation[_3D] = rigid3DLandmarkRegistration(landmarks, origin3D)
+  /**
+   * Returns a rigid transformation mapping the original landmarks  (first elements of the tuples) into the corresponding target points (second elements of the tuples).
+   *
+   * @param landmarks sequence of corresponding landmarks
+   * @param center center of rotation to be used for the rigid transformation
+   */
 
-  def similarity3DLandmarkRegistration(landmarks: IndexedSeq[(Point[_3D], Point[_3D])]): ParametricTransformation[_3D] = similarity3DLandmarkRegistration(landmarks, origin3D)
+  def rigid2DLandmarkRegistration(landmarks: Seq[(Point[_2D], Point[_2D])], center: Point[_2D]): RigidTransformation[_2D] = {
+    val (t, rotparams, _) = rigidSimilarity2DCommon(landmarks, center)
+    val optimalParameters = DenseVector.vertcat(t, DenseVector(Array(rotparams)))
+    val rigidSpace = RigidTransformationSpace[_2D](center)
+    rigidSpace.transformForParameters(optimalParameters)
+  }
+
+  /**
+   * Returns a rigid transformation mapping the original landmarks (first elements of the tuples) into the corresponding target points (second elements of the tuples).
+   *
+   * @param landmarks sequence of corresponding landmarks
+   * @param center - center of the rotation
+   */
+  def rigid2DLandmarkRegistration(originalLms: Seq[Landmark[_2D]], targetLms: Seq[Landmark[_2D]], center: Point[_2D]): RigidTransformation[_2D] = {
+    val commonLmNames = targetLms.map(_.id) intersect originalLms.map(_.id)
+    val landmarksPairs = commonLmNames.map(name => (originalLms.find(_.id == name).get.point, targetLms.find(_.id == name).get.point))
+    LandmarkRegistration.rigid2DLandmarkRegistration(landmarksPairs.toIndexedSeq, center)
+  }
+
+  /**
+   * Returns a similarity transformation mapping the original landmarks (first elements of the tuples)
+   * into the corresponding target points (second elements of the tuples).
+   *
+   * Attention: the center of rotation used for the rigid transformation here is the origin of the 3D space
+   *
+   * @param landmarks sequence of corresponding landmarks
+   */
+  def similarity3DLandmarkRegistration(landmarks: Seq[(Point[_3D], Point[_3D])]): ParametricTransformation[_3D] = similarity3DLandmarkRegistration(landmarks, origin3D)
```

Comment:
Same comment as above, the center or rotation is defaulted here which does not match with the strategy to make it mandatory.